share/workspaces/Advanced.xml

```diff
@@ -1589,7 +1589,7 @@
             <text>mute</text>
             </StaffText>
           </Cell>
-        <Cell name="Open" trElement="1">
+        <Cell name="Open (Text)" trElement="1">
```

Comment:
I don't like "Text" here, but am struggling to fine a better alternative. Something that gets the meanong across better, this being the opposite of "mute"

Reply:
Why changing it in the first place, is there a collision/duplicate? Where, and can't that get changed?

Reply:
Yes, there is a collision with the "open" symbol in the Articulations palette. I wrote "text" here to show that this one is not a symbol, and is actually written down as text.

Reply:
or "Open (Articulation)"
But isn't there also a "Mute" articulation, which would collide with the "Mute" text?
Ah, no, I see that is "Muted (closed)", so maybe changing the corresponding articulation "Open" to "Opened"?

Reply:
Yeah, I think "Muted" and "Opened" sound good. I'll change it.

Reply:
The articulation showing how it is set, (adjective) for that one note, the channel switch text showing how it should get set (imparativ) from this point on. I like it